### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by ayushinav**
* [x] Home
* [x] FNO tutorial
* [x] DeepONet tutorial
* [x] NOMAD tutorial
* [x] Kernel and Conv layers
* [ ] Benchmark (?)

### CODE REVIEW

**Comment by avik-pal** [comment]
@ayushinav lets try to wrap up this PR. Docs and tutorials should allow us to register this version of NeuralOperators

File: docs/.JuliaFormatter.toml
Comment (avik-pal) [inline docs/.JuliaFormatter.toml:1]: not needed, it should pickup the one in the top-level directory

**Comment by ayushinav** [comment]
We need the documenter key [here](https://github.com/ayushinav/NeuralOperators.jl/blob/docs/.buildkite/pipeline.yml#L45)

**Comment by avik-pal** [comment]
This doesn't need a buildkite deployment. GH-actions is fine here

**Pull request merged**